examples/deployment.cfg

```diff
@@ -23,9 +23,16 @@ libvirt_endpoint = qemu:///system
 
 [driver simple_vlan]
 switch = {"switch": "dell", "ip": "192.168.0.1", "user": "foo", "pass": "bar"}
-# The port that the switch is trunking to the head node. Note that this is
-# currently set manually, but must be known for deployment testing.
-trunk_port = gi1/0/4
+# A comma separated list of ports which may be manually configured to be trunked
+# on HaaS's vlans. The deployment tests need this information when verifying
+# that their configuration changes had the expected effect.
+#
+# Ports may be set up manually this way for a few different reasons, including:
+#
+# * The port is used to give headnodes access to the network
+# * The port is used to bridge networks to another switch (as used by the
+#   complex vlan driver).
+trunk_ports = gi1/0/4, gi1/0/5
```

Comment:
Quoting henn (2015-05-22 17:36:14)

> > +# * The port is used to give headnodes access to the network
> > +# * The port is used to bridge networks to another switch (as used by the
> > +#   complex vlan driver).
> > +trunk_ports = gi1/0/4, gi1/0/5
> 
>    What is this used for anymore?

Good call, it isn't --- got obsoleted when I changed the way we were 
querying the switch. I'll take it out.